Mathlib/Topology/Algebra/Field/Subfield.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2022 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Xavier Roblot
+-/
+import Mathlib.Algebra.Field.Subfield
+import Mathlib.Topology.Algebra.Field.Basic
+
+/-!
+# Topological properties of subfields
+
+This file constructs the topological closure of a subfield as a subfield.
+-/
+
+namespace Subfield
+variable {α : Type*} [Field α] [TopologicalSpace α] [TopologicalDivisionRing α]
+
+/-- The (topological-space) closure of a subfield of a topological field is
+itself a subfield. -/
+def topologicalClosure (K : Subfield α) : Subfield α where
```

Comment:
Note that `Subring.topologicalClosure` is not split to its own file; so this doesn't seem very beneficial to me either.